mixer/cmd/root.go

```diff
@@ -49,6 +49,15 @@ var RootCmd = &cobra.Command{
 			}
 		}
 
+		if rootCmdFlags.logFile != "" {
+			// Configure logger
+			_, err := log.SetOutputFilename(rootCmdFlags.logFile)
+			if err != nil {
+				failf("couldn't create file for log: %s", err)
```

Comment:
I think we should not terminate if there is an error in setting the log file. We should just print a warning to stdout.